### PR TITLE
Add Kubernetes attributes table & update example

### LINKED ISSUE #76: Add allowVolumeExpansion to storage class documentation
Page: [Kubernetes and Trident objects](https://docs.netapp.com/us-en/trident/trident-reference/objects.html)

https://docs.netapp.com/us-en/trident/trident-reference/objects.html#kubernetes-attributes

Can we please add  allowVolumeExpansion to the list of Kubernetes attributes? This has been part of CSI since K8s 1.16 and I'd consider it a best practise to add this to storage class definitions.

| Attribute  | Type | Values | Description | Relevant Drivers | Kubernetes Version
| ------------- | ------------- | ------------- | ------------- | ------------- | ------------- |
| allowVolumeExpansion  | boolean | true, false |  Enable or disable support for growing the PVC size | ontap-nas, ontap-nas-economy, ontap-nas-flexgroup, ontap-san, solidfire-san, aws-cvs, gcp-cvs, azure-netapp-files | 1.16+

Please see https://docs.netapp.com/us-en/trident/trident-use/vol-expansion.html for further details on volume expansion.

Also, can we please add this attribute to the example YAML manifest for a storage class shown on this page?

### PR DESCRIPTION
**Pull request by juliantap**
Add attributes to Kubernetes attributes table & updated YAML example to close #76

### CODE REVIEW

File: trident-reference/objects.adoc [hunk 1]
```diff
@@ -184,8 +186,12 @@ These attributes have no impact on the selection of storage pools/backends by Tr
 Version
 |fsType |string |ext4, ext3, xfs, etc. |The file system type for block
 volumes |solidfire-san, ontap-san, ontap-san-economy, eseries-iscsi |All
+|allowVolumeExpansion |boolean |true, false |Enable or disable support for growing the PVC size |ontap-nas, ontap-nas-economy, ontap-nas-flexgroup, ontap-san, solidfire-san, aws-cvs, gcp-cvs, azure-netapp-files |1.6+
```
Comment (balaramesh) [inline trident-reference/objects.adoc:189]: Version must be 1.11+

File: trident-reference/objects.adoc [hunk 2]
```diff
@@ -184,8 +186,12 @@ These attributes have no impact on the selection of storage pools/backends by Tr
...
+|volumeBindingMode |string |Immediate, WaitForFirstConsumer |Choose when volume binding and dynamic provisioning occurs |ontap-nas, ontap-nas-economy, ontap-nas-flexgroup, ontap-san, solidfire-san, aws-cvs, gcp-cvs, azure-netapp-files |1.6+
```
Comment (balaramesh) [inline trident-reference/objects.adoc:190]: Version must be 1.17+ here

**Review by balaramesh — CHANGES_REQUESTED** [review]

File: trident-reference/objects.adoc [hunk 3]
```diff
@@ -183,10 +185,25 @@ These attributes have no impact on the selection of storage pools/backends by Tr
 |Attribute |Type |Values |Description |Relevant Drivers |Kubernetes
 Version
 |fsType |string |ext4, ext3, xfs, etc. |The file system type for block
-volumes |solidfire-san, ontap-san, ontap-san-economy, eseries-iscsi |All
+volumes |solidfire-san, ontap-nas, ontap-nas-economy, ontap-nas-flexgroup, ontap-san, ontap-san-economy, eseries-iscsi |All
+|allowVolumeExpansion |boolean |true, false |Enable or disable support for growing the PVC size |ontap-nas, ontap-nas-economy, ontap-nas-flexgroup, ontap-san, solidfire-san, aws-cvs, gcp-cvs, azure-netapp-files |1.11+
+|volumeBindingMode |string |Immediate, WaitForFirstConsumer |Choose when volume binding and dynamic provisioning occurs |ontap-nas, ontap-nas-economy, ontap-nas-flexgroup, ontap-san, solidfire-san, aws-cvs, gcp-cvs, azure-netapp-files |1.17+
```
[the remark this answers is not in the document]
Comment (juliantap) [reply trident-reference/objects.adoc]: Done! Thanks! I'll merge now @balaramesh

**Review by balaramesh — APPROVED** [review]
Thanks! LGTM

**Pull request merged**